Tasks/Lesson_5/Methodichka_11.java

```diff
@@ -0,0 +1,23 @@
+package Tasks.Lesson_5;
+
+/*Имеется целое число, определить является ли это число простым, т.е.
+делится без остатка только на 1 и себя.*/
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Methodichka_11 {
+    public static void main(String[] args) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("Enter the number: ");
+        int number = Integer.parseInt(reader.readLine());
+        int count = 0;
+        for (int i = 2; i < number; i++) {
```

Comment:
ok. But cycle iterations could be less than from 2 to number. At least from 0 to number/2 (or even to sqrt(number))